wger/core/views/user.py

```diff
@@ -323,6 +330,82 @@ def preferences(request):
         return render(request, 'user/preferences.html', template_data)
 
 
+@login_required
+def sync_fitbit(request):
+    """Authorizes the connection with Wger
+
```

Comment:
The formatting here is a bit off. You can either bring the lower triple quotes up to line 335 or take the `Authorizes the connection with Wger` down to line 336.

Reply:
> The formatting here is a bit off. You can either bring the lower triple quotes up to line 335 or take the `Authorizes the connection with Wger` down to line 336.

Noted.